node_modules/gh-users/tests/test-terms-and-conditions.js

```diff
@@ -0,0 +1,296 @@
+                        assert.strictEqual(me.termsAndConditions.needsToAccept, true);
+
+                        // Accept the Terms and Conditions
+                        UsersTestsUtil.assertAcceptTermsAndConditions(simon.client, simon.profile.id, function() {
+
+                            // Verify the me feed confirms that we don't need to accept the Terms and Conditions anymore
+                            UsersTestsUtil.assertGetMe(simon.client, function(me) {
+                                assert.strictEqual(me.termsAndConditions.needsToAccept, false);
+
+                                // Verify we can interact with the system
+                                SeriesTestsUtil.assertCreateSerie(simon.client, 'Test serie', null, function() {
+
+                                    // Update the Terms and Conditions
+                                    var configuration = {
+                                        'termsAndConditionsText': 'Oh heavens no'
+                                    };
+                                    ConfigTestsUtil.assertUpdateConfig(globalAdminClient, app.id, configuration, function() {
+
+                                        // Verify the user needs to re-accept the Terms and Conditions
+                                        UsersTestsUtil.assertGetMe(simon.client, function(me) {
+                                            assert.strictEqual(me.termsAndConditions.needsToAccept, true);
+                                            return callback();
+                                        });
+                                    });
+                                });
+                            });
+                        });
+                    });
+                });
+            });
+        });
+
+        /**
+         * Test that verifies that anonymous users don't need to accept the Terms and Conditions
+         */
+        it('verify anonymous users don\'t need to accept the Terms and Conditions', function(callback) {
+            TestsUtil.generateTestTenant(1, function(tenant, app) {
+
+                // Set some Terms and Conditions
+                TestsUtil.getGlobalAdminRestClient(function(globalAdminClient) {
+                    var configuration = {
+                        'termsAndConditionsEnabled': true,
+                        'termsAndConditionsText': 'Oh lawdy'
+                    };
+                    ConfigTestsUtil.assertUpdateConfig(globalAdminClient, app.id, configuration, function() {
+
+                        // Verify anonymous users do not have to accept them
+                        TestsUtil.getAnonymousAppUserClient(app, function(anonymousClient) {
+                            UsersTestsUtil.assertGetMe(anonymousClient, function(me) {
+                                assert.strictEqual(me.termsAndConditions.needsToAccept, false);
+                                return callback();
```

Comment:
Just confirming that there are no POST requests in the system that an anonymous user should be able to submit that we can verify here?